native/sapp-wasm/src/lib.rs

```diff
@@ -252,7 +252,7 @@ pub unsafe fn sapp_run(desc: *const sapp_desc) -> ::std::os::raw::c_int {
             let msg = CString::new(format!("{:?}", info)).unwrap_or_else(|_| {
                 CString::new(format!("MALFORMED ERROR MESSAGE {:?}", info.location())).unwrap()
             });
-            test_log(msg.as_ptr());
+            raw_console_log(msg.as_ptr());
```

Comment:
lets keep it that way until we finish this, but later we can just use `log!()` here, without converting to CString and `#[cfg]`

Reply:
I don't think `log!()` will be available here, will it? Since we're implementing it at the `sapp-console-log` level, as a dependency of `gwg`?